app/jobs/inboxes/fetch_imap_emails_job.rb

```diff
@@ -2,6 +2,8 @@ class Inboxes::FetchImapEmailsJob < ApplicationJob
   queue_as :low
 
   def perform(channel)
+    return unless channel.imap_enabled?
```

Comment:
please remove '?'

`return unless channel.imap_enabled`

Reply:
@aswindevps `channel.imap_enabled?` also works as `channel.imap_enabled` it's a ruby convention to write it with a question mark at the end if it's returning a boolean value.

Reply:
this should be okay. ref: https://stackoverflow.com/a/1345854/939299